dev/reference/tools/data-migration/dm-portal.md

```diff
@@ -0,0 +1,235 @@
+---
+title: DM Portal 简介
+category: reference
+---
+
+# DM Portal 简介
+
+当前版本的 DM 的提供了丰富多样的功能特性，包括 [Table routing](/reference/tools/data-migration/features/overview.md#table-routing)，[Black & white table lists](/reference/tools/data-migration/features/overview.md#black-white-table-lists)，[Binlog event filter](/reference/tools/data-migration/features/overview.md#binlog-event-filter)，[Column mapping](/reference/tools/data-migration/features/overview.md#column-mapping) 等。但这些功能特性同时也增加了用户使用 DM 的复杂度，尤其在编写 [DM 任务配置](/reference/tools/data-migration/configure/task-configuration-file/) 时候。
+针对这个问题，我们提供了一个非常精简的小网页程序  DM Portal，能够帮助用户以可视化的方式去配置需要的同步任务，并且生成可以直接让 DM 直接执行的 task.yaml 文件。
+
+## 功能描述
+
+### 同步模式配置
+
+支持 DM 的三种同步模式：
+
+- 全量同步
+- 增量同步
+- All(全量+增量）
+
+### 实例信息配置
+
+支持配置库表同步路由方式，能够支持 DM 中分库分表合并的配置方式。
+
+### binlog 过滤配置
+
+支持对数据库、数据表配置 binlog event 过滤。
+
+### 配置文件生成
+
+支持配置文件创建，能够将配置文件下载到本地并且同时会在 dm-portal 服务器 /tmp/ 目录下自动创建。
+
+### 使用限制
+
+当前的 DM 配置可视化生成页面能够覆盖绝大部分的 DM 配置场景，但还是有一定的使用限制：
+
+- 不支持 [Column mapping](/reference/tools/data-migration/features/overview.md#column-mapping)
+- 不支持 [Binlog event filter](/reference/tools/data-migration/features/overview.md#binlog-event-filter) 的 sql pattern 方式
+- 编辑功能不支持解析用户之前写的 task.yaml 文件，页面只能编辑由页面生成的 task.yaml 文件
+- 编辑功能不支持修改实例配置信息，如果用户需要调整实例配置，需要重新生成 task.yaml 文件
+- 页面的上游实例配置仅用于获取上游库表结构，dm-worker 里依旧需要配置对应的上游实例信息
+- 生成的 task.yaml 中默认 mydumper-path 为 "./bin/mydumper"，如果实际使用其他路径，需要在生成的配置文件中进行手动修改。
+
+## 部署使用
+
+### binarey 部署
+
+DM Portal 可以在 [dm-portal-latest.tar.gz](https://download.pingcap.org/dm-portal-latest.tar.gz) 下载，通过 `./dm-portal` 的命令就能直接启动。
+
+* 如果在本地启动，浏览器访问 127.0.0.1:8280 就能使用。
```

Comment:
```suggestion
* 如果在本地启动，浏览器访问 `127.0.0.1:8280` 即可使用。
```

Reply:
Fix in https://github.com/pingcap/docs-cn/pull/1679/commits/998f85400384502d92a0435fca5529c0d21b71fa